handlers/bot_handler.py

```diff
@@ -54,14 +54,36 @@ def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_ad
             raise InvalidCommand("Sorry, couldn't retrieve the git information for the bot...")
 
 
+class InviteCommand(Command):
+    """
+    Invite a list of members to the current channel, ignores members already
+    present.
+    """
+
+    @classmethod
+    def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_admin):
+        try:
+            current_members = slack_wrapper.get_channel_members(channel_id)
+            # strip uid formatting
+            invited_users = [user.strip("<>@") for user in args]
+            # remove already present members
+            invited_users = [user for user in invited_users if user not in current_members]
+            for member in invited_users:
+                if not slack_wrapper.invite_user(member, channel_id)["ok"]:
+                    raise SlackClientError
+        except SlackClientError:
+            log.exception("BotHandler::InviteCommand")
+            raise InvalidCommand("Sorry, couldn't invite the given members to the channel")
+
 class BotHandler(BaseHandler):
     """Handler for generic bot commands."""
 
     def __init__(self):
         self.commands = {
             "ping": CommandDesc(PingCommand, "Ping the bot", None, None),
             "intro": CommandDesc(IntroCommand, "Show an introduction message for new members", None, None),
-            "version": CommandDesc(VersionCommand, "Show git information about the running version of the bot", None, None)
+            "version": CommandDesc(VersionCommand, "Show git information about the running version of the bot", None, None),
+            "invite": CommandDesc(InviteCommand, "Invite a list of members to the current channel (smarter than /invite)", None, None)
```

Comment:
The invite command should have a mandatory argument `user_list` (describing that you have to pass a user or a list of users separated by whitespace).

Also might be worth noting that users have to be passed in at notation.